### ISSUE TITLE
IMathics vs IWolfram

### ISSUE THREAD

**Issue by GokulNC** [mmatera/iwolfram#12]
Sorry if this question is dumb, I nowhere found the answers anywhere.

How is this repo (iwolfram) different from [imathics](https://github.com/mathics/IMathics) (by Mathics) ? (except having support for Wolfram kernel)

Does it include all features implemented from IMathics?
If I'm using Mathics, which one would be better to use?

It'd be great if you could add these details somewhere in the repo.
Thanks a lot for this repo BTW :)

**Comment by Kreijstal** [comment]
what is IMathics?

**Comment by mmatera** [comment]
Indeed, I need to work on the documentation, and in many other things...

There are two different projects: Iwolfram (this) and IMathics.
The IWolfram approach is to define a jupyter kernel based on Metakernel, that interacts with a command line interpreter of WL, and hence, works both with Wolfram Mathematica, Mathics and any other implementation of the language.

IMathics has a different approach: Mathics is a implemenation of WL in Python, which can interact directly with the mathics library (without dealing with a command line interpreter). This approach just works with mathics.

In a similar line, Wolfram Research released recently their own jupyter kernel,  https://github.com/WolframResearch/WolframLanguageForJupyter  with of course, just works with Mathematica, but not with Mathics or other implementations.
